src/utils/gaa.js

```diff
@@ -694,6 +701,30 @@ function callSwg(callback) {
   (self.SWG = self.SWG || []).push(callback);
 }
 
+/**
+ * Logs Showcase events.
+ * @param {!ShowcaseEvent} showcaseEvent
+ */
+function logEvent(showcaseEvent) {
+  callSwg((swg) => {
+    // Get reference to event manager.
+    swg.getEventManager().then((eventManager) => {
+      // Get individual analytics events from Showcase event.
+      const eventTypes = showcaseEventToAnalyticsEvents(showcaseEvent);
+
+      // Log each analytics event.
+      eventTypes.forEach((eventType) => {
+        eventManager.logEvent({
+          eventType,
+          eventOriginator: EventOriginator.SWG_CLIENT,
```

Comment:
Just double checking in case this should be `EventOriginator.SHOWCASE_CLIENT`?

Reply:
Yep! `src/runtime/entitlements-manager.js` suggests `SHOWCASE_CLIENT` represents publishers